src/tests/ftest/osa/osa_offline_reintegration.py

```diff
@@ -30,97 +24,63 @@ def setUp(self):
-
-        # Add a thread for these IOR arguments
-        process = threading.Thread(target=self.ior_thread,
-                                   kwargs={"pool": self.pool,
-                                           "oclass": oclass,
-                                           "api": api,
-                                           "test": test,
-                                           "flags": flags,
-                                           "results":
-                                           self.out_queue})
-        # Launch the IOR thread
-        process.start()
-        # Wait for the thread to finish
-        process.join()
-
 
     def run_offline_reintegration_test(self, num_pool, data=False,
-                                       server_boot=False):
+                                       server_boot=False, oclass=None):
         """Run the offline reintegration without data.
             Args:
             num_pool (int) : total pools to create for testing purposes.
             data (bool) : whether pool has no data or to create
                           some data in pool. Defaults to False.
             server_boot (bool) : Perform system stop/start on a rank.
                                  Defults to False.
+            oclass (str) : daos object class string (eg: "RP_2G8")
         """
         # Create a pool
         pool = {}
         pool_uuid = []
-        exclude_servers = (len(self.hostlist_servers) * 2) - 1
 
-        # Exclude rank : two ranks other than rank 0.
-        rank = random.randint(1, exclude_servers)
+        # Exclude ranks [0, 3, 4]
+        rank = [0, 3, 4]
+        if oclass is None:
+            oclass = self.ior_dfs_oclass[0]
 
         for val in range(0, num_pool):
             pool[val] = TestPool(self.context,
                                  dmg_command=self.get_dmg_command())
             pool[val].get_params(self)
-            # Split total SCM and NVME size for creating multiple pools.
-            pool[val].scm_size.value = int(pool[val].scm_size.value /
-                                           num_pool)
-            pool[val].nvme_size.value = int(pool[val].nvme_size.value /
-                                            num_pool)
             pool[val].create()
             pool_uuid.append(pool[val].uuid)
             self.pool = pool[val]
             if data:
-                self.run_ior_thread("Write", self.ior_dfs_oclass[0],
-                                    self.ior_apis[0], self.ior_test_sequence[0])
-
-        # Exclude and reintegrate the pool_uuid, rank and targets
-        for val in range(0, num_pool):
-            self.pool = pool[val]
+                self.run_ior_thread("Write", oclass,
+                                    self.ior_apis[0],
+                                    self.ior_test_sequence[0])
+
+        # Exclude all the ranks
+        random_pool = random.randint(0, (num_pool-1))
+        for val in range(len(rank)):
```

Comment:
(pylint-consider-using-enumerate) Consider using enumerate instead of iterating with range and len